modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/home/HomePage.java

```diff
@@ -50,4 +52,12 @@ public ManagementConsoleLoginPage logout() throws IOException {
         driver.findElement(By.xpath(uiElementMapper.getElement("home.greg.sign.out.xpath"))).click();
         return new ManagementConsoleLoginPage(driver);
     }
+
```

Comment:
Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158